src/paradigma/config.py

```diff
@@ -0,0 +1,408 @@
+        super().__init__()
+
+        self.classifier_file_name = "gait_filtering_classifier.pkl"
+        self.thresholds_file_name = "gait_filtering_threshold.txt"
+
+        self.set_filenames('arm_activity')
+
+
+class TremorDetectionConfig(TremorBaseConfig):
+
+    def __init__(self) -> None:
+        super().__init__()
+
+        self.coefficients_file_name = "tremor_detection_coefficients.txt"
+        self.thresholds_file_name = "tremor_detection_threshold.txt"
+        self.mean_scaling_file_name = "tremor_detection_mean_scaling.txt"
+        self.std_scaling_file_name = "tremor_detection_std_scaling.txt"
+
+        self.fmin_peak: float = self.fmin_tremor_power
+        self.fmax_peak: float = self.fmax_tremor_power
+
+        self.d_channels_values = {
+            DataColumns.PRED_TREMOR_PROBA: "probability",
+            DataColumns.PRED_TREMOR_LOGREG: "boolean",
+            DataColumns.PRED_TREMOR_CHECKED: "boolean"
+        }
+
+        self.set_filenames('tremor')
+
+
+class SignalQualityClassificationConfig(HeartRateBaseConfig):
+
+    def __init__(self) -> None:
+        super().__init__()
+        self.classifier_file_name = "ppg_quality_classifier.pkl"
+        self.thresholds_file_name = "ppg_acc_quality_threshold.txt"
+
+        self.set_filenames('gait')
+
+
+# Quantification
+class ArmSwingQuantificationConfig(GaitBaseConfig):
+
+    def __init__(self) -> None:
+        super().__init__()
+
+        self.min_segment_length_s = 3
+
+
+class TremorQuantificationConfig(TremorBaseConfig):
+    def __init__(self) -> None:
+        super().__init__()
+
+        self.valid_day_threshold_hr: float = 0 # change to 10 later!
+        self.daytime_hours_lower_bound: float = 8
+        self.daytime_hours_upper_bound: float = 22
+        self.percentile_tremor_power: float = 0.9
+
+        self.set_filenames('tremor')
+        
+
+class HeartRateExtractionConfig(HeartRateBaseConfig):
+
+    def __init__(self, min_window_length: float = 10) -> None:
+        super().__init__()
+
+         # Parameters for HR analysis
+        self.window_length_s: int = 6
+        self.window_step_size_s: int = 1
+        self.min_hr_samples = min_window_length * self.sampling_frequency
+        self.threshold_sqa = 0.5
+
+        # Heart rate estimation parameters
+        hr_est_length = 2
+        self.hr_est_samples = hr_est_length * self.sampling_frequency
+
+        # Time-frequency distribution parameters
+        self.tfd_length = 10
+        self.kern_type = 'sep'
+        win_type_doppler = 'hamm'
+        win_type_lag = 'hamm'
+        win_length_doppler = 1
+        win_length_lag = 8
+        doppler_samples = self.sampling_frequency * win_length_doppler
+        lag_samples = win_length_lag * self.sampling_frequency
+        self.kern_params = [
+            {'doppler_samples': doppler_samples, 'win_type_doppler': win_type_doppler}, 
+            {'lag_samples': lag_samples, 'win_type_lag': win_type_lag}
+        ]
```

Comment:
Deze laatste (heartrateextractionconfig) update ik in mijn volgende PR maar dit mag je wel eerst mergen en dan pas ik hem vervolgens aan.